src/main/java/org/broadinstitute/consent/http/models/DatasetProperty.java

```diff
@@ -10,24 +11,59 @@ public class DatasetProperty {
     private Integer dataSetId;
     private Integer propertyKey;
     private String propertyName;
-    private String propertyValue;
+    private Object propertyValue;
     private Date createDate;
+    private String schemaProperty;
+    private DatasetPropertyType propertyType;
 
     public DatasetProperty(){
     }
 
-    public DatasetProperty(Integer propertyId, Integer  dataSetId, Integer propertyKey, String propertyValue,
+    public DatasetProperty(Integer propertyId,
```

Comment:
should these constructors which do not have the schemaProperty field be deprecated?

Reply:
Yes, we should either refactor or deprecate them